WalletWasabi.Fluent/Helpers/ClipboardHelper.cs

```diff
@@ -0,0 +1,51 @@
+using System.Reactive.Concurrency;
+using System.Reactive.Linq;
+using NBitcoin;
+using WalletWasabi.Fluent.Extensions;
+
+namespace WalletWasabi.Fluent.Helpers;
+
+internal class ClipboardHelper
```

Comment:
You're right. I chose the "Helper" suffix just because I felt nothing in our convention really adapter better. 

In regards to its location, I still don't know if the Models folder makes sense. In a domain-driven approach, model classes would be classes that have a direct meaning into "our business". "Wallet", "Money", "Transaction"... but in our architecture, I still don't know what the Model folder is for, to be honest. In my personal projects I usually have folders for infrastructure abstractions like this. 

Anyways, I'm fine with "Models" as long as we all know what to put in there.

Reply:
Yeah I feel the same, I am not sure either.

>  In my personal projects I usually have folders for infrastructure abstractions like this.

I think this makes sense. Let's do this.